### PR TITLE
Possibility to specify index when uploading data for the timesketch_api_client

### PR DESCRIPTION
**Pull request by benua74**
Hi,
I have slighty modified the code of the timesketch api client in order to be able to specify an arbitrary index instead of the default uuid.
I kept the default behavior :)

if you find it useful.

### CODE REVIEW

**Comment by benua74** [comment]
@googlebot I signed it!

File: api_client/python/timesketch_api_client/sketch.py [hunk 1]
```diff
@@ -249,7 +249,7 @@ def upload(self, timeline_name, file_path):
         """
```
Comment (kiddinn) [inline api_client/python/timesketch_api_client/sketch.py]: missed added the index into the Args section

**Comment by kiddinn** [comment]
also, it seems like there are some issues with the CLA,

File: api_client/python/timesketch_api_client/sketch.py [hunk 2]
```diff
@@ -249,7 +249,7 @@ def upload(self, timeline_name, file_path):
...
         resource_url = '{0:s}/upload/'.format(self.api.api_root)
         files = {'file': open(file_path, 'rb')}
-        data = {'name': timeline_name, 'sketch_id': self.id}
+        data = {'name': timeline_name, 'sketch_id': self.id, 'index_name': index}
```
Comment (kiddinn) [inline api_client/python/timesketch_api_client/sketch.py]: api_client/python/timesketch_api_client/sketch.py:252:0: C0301: Line too long (81/80) (line-too-long)

the linter is complaining here,

**Comment by kiddinn** [comment]
I also didn't hit the "update branch" command, you may want to do that since the branch is out of date with the base branch... once you do that, you'll have to do a `git pull` before making changes in order to avoid a conflict.

**Comment by kiddinn** [comment]
I'm approving this, but #1099 will change how this is done, but will add this functionality into that PR

**Comment by benua74** [comment]
> I'm approving this, but #1099 will change how this is done, but will add this functionality into that PR

ok, great !

**Pull request merged**